sa-ma/scorer
# #168494964 Set up Redux

**What this PR does?**
- setup redux
- setup redux-thunk
- setup redux-devtools

**How to test manually**
- Clone repo
- Run `npm install`
- Run `npm start`
- Open Google Chrome 
- Download and install Redux Devetools Extension from [Chrome Web Store](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd)
- Open the Developers Console and click on the redux tab to see the redux state 

**What are the Relevant Pivotal Tracker Stories?**
[#168494964](https://www.pivotaltracker.com/story/show/168494964)
